pom.xml

```diff
@@ -217,7 +217,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
     <objenesis.version>1.0</objenesis.version>
     <okhttp.version>2.7.5</okhttp.version>
     <okio.version>3.6.0</okio.version>
-    <mockito2.version>3.5.9</mockito2.version>
+    <mockito3.version>3.5.9</mockito3.version>
```

Comment:
Well spotted.

This was originally named `mockito2.version` to distinguish from the version property for Mockito 1, which was also a dependency at the time.  I think now we can rename this to simply `mockito.version`.

Reply:
done